src/modlunky2/ui/trackers/runstate.py

```diff
@@ -187,13 +200,15 @@ def update_has_mounted_tame(self, player_overlay):
         if self.theme == Theme.TIAMAT and entity_type == EntityType.MOUNT_QILIN:
             self.lc_has_mounted_qilin = True
             self.failed_low_if_not_chain = True
+            if not self.is_chain:
+                self.run_label.discard(Label.LOW)
```

Comment:
same question here about why not `self.fail_low()`

Reply:
Good catch. Fixed both places